cpp/tests/filling/filling_tests.cu

```diff
@@ -102,6 +107,36 @@ TYPED_TEST(FillingTest, SetRange)
   FillTest(begin, end, val, false);
 }
 
+TYPED_TEST(FillingTest, SetRangeNullCount)
+{
+  gdf_index_type begin = 10;
+  gdf_index_type end = 50;
+  TypeParam val = TypeParam{1};
+
+  column_validity_fn_t some_valid = [](gdf_index_type row) { 
+    return row > 500 ? false : true;
```

Comment:
What if the column size is < 500? This test will be meaningless. Just use odd and even rows or something.